testing/test_runner.py

```diff
@@ -1094,3 +1094,20 @@ def func() -> None:
     with pytest.raises(TypeError) as excinfo:
         OutcomeException(func)  # type: ignore
     assert str(excinfo.value) == expected
+
+
+def test_pytest_version_env_var(pytester: Pytester, monkeypatch: MonkeyPatch) -> None:
+    os.environ["PYTEST_VERSION"] = "old version"
```

Comment:
Shouldn’t `monkeypatch.setenv()` be used to ensure it’s reset after this test?

https://docs.pytest.org/en/latest/how-to/monkeypatch.html#monkeypatching-environment-variables

Reply:
Ahh good catch, missed that. :+1: 

Anyone willing to open a quick PR to fix that?

Reply:
https://github.com/pytest-dev/pytest/pull/12225
@nicoddemus